backend/services/news/src/datasources/News.ts

```diff
@@ -95,17 +97,24 @@ export default class News extends dbUtils.KnexDataSource {
 
       const uploadUrl = await getUploadUrl(articleInput.imageName);
 
-      const newArticle = {
+      const newArticle: any = {
         header: articleInput.header,
         header_en: articleInput.headerEn,
         body: articleInput.body,
         body_en: articleInput.bodyEn,
-        author_id: user.member_id,
         published_datetime: new Date(),
         image_url: uploadUrl?.fileUrl,
       };
-      const id = (await this.knex<sql.Article>('articles').insert(newArticle).returning('id'))[0];
-      const article = { id, ...newArticle };
+
+      if (articleInput.mandateId) {
```

Comment:
If someone knew what they were doing and found out someone elses mandateId couldn't they technically put out an article in someone elses name? We should have some kind of validation that the mandateId belongs to them.

Reply:
Technically yes, but you would need access to creating or updating news and would need malicious intent, which I find unlikely. I could add the check anyway, though!

Reply:
Perhaps the best solution is to have a member field that saves the member id of the creator of the article? We should really add this in more places so that we can trace which member did what.